src/vs/workbench/contrib/debug/browser/disassemblyView.ts

```diff
@@ -0,0 +1,209 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { getPixelRatio, getZoomLevel } from 'vs/base/browser/browser';
+import { Dimension, append, $ } from 'vs/base/browser/dom';
+import { ITableRenderer, ITableVirtualDelegate } from 'vs/base/browser/ui/table/table';
+import { BareFontInfo } from 'vs/editor/common/config/fontInfo';
+import { localize } from 'vs/nls';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
+import { WorkbenchTable } from 'vs/platform/list/browser/listService';
+import { IStorageService } from 'vs/platform/storage/common/storage';
+import { ITelemetryService } from 'vs/platform/telemetry/common/telemetry';
+import { editorBackground } from 'vs/platform/theme/common/colorRegistry';
+import { IThemeService } from 'vs/platform/theme/common/themeService';
+import { EditorPane } from 'vs/workbench/browser/parts/editor/editorPane';
+import { EditorInput } from 'vs/workbench/common/editor/editorInput';
+import { DISASSEMBLY_VIEW_ID } from 'vs/workbench/contrib/debug/common/debug';
+
+interface IDisassembledInstructionEntry {
+	allowBreakpoint: boolean;
+	isBreakpointSet: boolean;
+	instruction: DebugProtocol.DisassembledInstruction;
+}
+
+export class DisassemblyView extends EditorPane {
+
+	private static readonly NUM_INSTRUCTIONS_TO_LOAD = 50;
+
+	private _fontInfo: BareFontInfo;
+	private _disassembledInstructions: WorkbenchTable<IDisassembledInstructionEntry> | null;
+
+
+	constructor(
+		@ITelemetryService telemetryService: ITelemetryService,
+		@IThemeService themeService: IThemeService,
+		@IStorageService storageService: IStorageService,
+		@IConfigurationService configurationService: IConfigurationService,
+		@IInstantiationService private readonly _instantiationService: IInstantiationService
+	) {
+		super(DISASSEMBLY_VIEW_ID, telemetryService, themeService, storageService);
+
+		this._fontInfo = BareFontInfo.createFromRawSettings(configurationService.getValue('editor'), getZoomLevel(), getPixelRatio());
+		configurationService.onDidChangeConfiguration(e => {
```

Comment:
Make sure to add this to a disposable store, so this listeners get disposed when you editor gets disposed.